website/content/en/preview/getting-started/scripts/step02-node-policies.sh

```diff
@@ -0,0 +1,11 @@
+aws iam attach-role-policy --role-name KarpenterInstanceNodeRole \
+    --policy-arn arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy
+
+aws iam attach-role-policy --role-name KarpenterInstanceNodeRole \
+    --policy-arn arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy
+
+aws iam attach-role-policy --role-name KarpenterInstanceNodeRole \
+    --policy-arn arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly
+
+aws iam attach-role-policy --role-name KarpenterInstanceNodeRole \
```

Comment:
this is not required so I would consider leaving this out. I will go out on a limb and say that I, like others, assumed it was required after seeing it in the guide

Reply:
Would deleting lines 7-10 take care of that then?

Reply:
yes

Reply:
I've deleted the whole step, per the suggestion in [b4ec84d](https://github.com/aws/karpenter/commit/b4ec84d65af52bc8c35cbdd871696577ad63e1ea)